Sources_v3/StreamChatUI/ChatChannel/MessageInputSlashCommandView.swift

```diff
@@ -4,20 +4,77 @@
 
 import UIKit
 
-open class MessageInputSlashCommandView: UIView {
-    // MARK: - Underlying types
-    
-    public enum Command: String {
-        case giphy
+public enum MessageComposerSuggestionCommandKind: Equatable {
+    case custom(identifier: String)
+    case ban
+    case flag
+    case giphy
```

Comment:
When `.custom(value)` appears in enum is the the sign to consider `RawRepresentable` + static values.  It is mostly relevant for the case when enum is `Codable` and custom `init(decoder)/encode(coder)` implementations are required but here it can also work well -> up to you 😄 

The `rawValue` will serve as `identifier`.

```suggestion
public struct MessageComposerSuggestionCommandKind: RawRepresentable {
   public let rawValue: String
   public init (rawValue: String) {
     self. rawValue = rawValue
   }
}

public extension MessageComposerSuggestionCommandKind {
 static let ban = Self(rawValue: "ban")
 ...
}
```

Reply:
Thanks, I actually removed the enum for good. We will have this implemented from LLC. Check the discussion on Slack channel.